src/startErrorHandling.js

```diff
@@ -1,33 +1,47 @@
 import { CONFIG, REDUCERS } from './constants';
-import Raven from 'raven-js';
+import * as Sentry from '@sentry/browser';
 import listenStoreChanges from './listenStoreChanges';
 
-const startErrorHandling = (store, raven = Raven, testing = false) => {
+const startErrorHandling = (store, sentry = Sentry, testing = false) => {
   if (!testing && CONFIG.env === 'test' || CONFIG.env === 'dev') {
     return;
   }
 
-  raven.config(CONFIG.ravenURL, { environment: CONFIG.env }).install();
+  sentry.init({
+    dsn: CONFIG.sentryURL,
+    environment: CONFIG.env,
+  });
 
   const addExtensionInfo = () => {
     if (!window.chrome || !window.chrome.management || !window.chrome.management.getSelf) {
       return;
     }
 
     window.chrome.management.getSelf(info => {
-      raven.setTagsContext({ 'extension.id': info.id, 'extension.version': info.version });
+      sentry.setTag('extension.id', info.id);
+      sentry.setTag('extension.version', info.version);
     });
   };
 
   const checkUUID = ({ worker: prevWorker }, { worker: curWorker }) => {
     if (!prevWorker.get('uuid') && curWorker.get('uuid')) {
-      raven.setUserContext({ uuid: curWorker.get('uuid') });
+      sentry.setUser({ id: curWorker.get('uuid') });
     }
   };
 
   const checkRelease = ({ plugin: prevPlugin }, { plugin: curPlugin }) => {
     if (prevPlugin.get('version') !== curPlugin.get('version')) {
-      raven.setRelease(curPlugin.get('version'));
+      sentry.configureScope(scope =>
+        scope.addEventProcessor(event => {
+          // eslint-disable-next-line no-new
+          new Promise(resolve => {
```

Comment:
I think you meant to return this promise. Right now this callback returns undefined so nothing is actually using the resolved value from the promise (also might mean this isn't tested well?)

I googled what I think you googled and in that example snippet, the callback is using parenthesis instead of brackets (`() => ()` instead of `() => {}`). With parens you can only evaluate a single expression in the function body, so that expression is implicitly returned.